contracts/reputationMiningCycle/ReputationMiningCycle.sol

```diff
@@ -534,6 +548,24 @@ contract ReputationMiningCycle is ReputationMiningCycleStorage, PatriciaTreeProo
     return reputationMiningWindowOpenTimestamp;
   }
 
+  function getDisputeRewardSize() public view returns (uint256) {
+    return disputeRewardSize();
```

Comment:
Okay, so the trouble here is in our EtherRouter setup scripts. By putting a public function in the `Common` file, it appears in the ABI of each contract, which the scripts balk at, claiming we've defined functions with the same name in multiple contracts (which, technically, we have). In theory, there's nothing wrong with that, but it's just a guard we've been keeping ourselves to to avoid confusions.

In theory, we could add an exception to any functions that appear in the Common ABI, but we'd have to change deployment scripts and so forth too, which use the same type of logic to check what's happening during deployment.

Reply:
No it's probably best not to mess with those conventions. I do like the idea of having it as a storage variable which is set when the submission window is closed.